build.gradle.kts

```diff
@@ -237,6 +237,8 @@ dependencies {
     implementation("org.javassist:javassist:3.28.0-GA")
     implementation("org.slf4j:slf4j-api:1.7.30")
     implementation("com.oracle.oci.sdk:oci-java-sdk-common:2.13.0")
+    compileOnly("com.amazonaws:aws-java-sdk-secretsmanager:1.12.213")
```

Comment:
These libraries belong to AWS SDK v.1.x, is there similar libraries for v.2.x? If they exist, we need to migrate to v.2.x

Reply:
I was able to find something for the secrets manager, but not the secrets manager caching.